CHANGELOG.md

```diff
@@ -92,6 +92,8 @@ External tooling making use of `searchContext` in the `GET` `/granules/` endpoin
   - Updated `@cumulus/api/bin/serveUtils` to no longer add records to ElasticSearch
   - Removed ElasticSearch from local API server code
   - Updated CollectionSearch to filter granule fields in addition to time frame for active collections
+- **CUMULUS-3847**
+  - list changes here
```

Comment:
?

Reply:
placeholder, I'm still making sure I've captured everything. will make sure this is filled out when I'm fairly certain

Reply:
Cool. I'll leave this comment unresolved for now.

Reply:
this has been updated, and the line removed as suggested